src/main/java/core/basesyntax/service/impl/FruitStorageServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitStorageService;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.List;
+
+public class FruitStorageServiceImpl implements FruitStorageService {
```

Comment:
It's a good practice to follow the encapsulation principle. Consider making the `FruitStorageServiceImpl` class fields `fruitStorageDao` and `operationStrategy` private to hide the internal state and protect the integrity of the instance.